docs/basics/Actions.md

```diff
@@ -53,30 +53,32 @@ Finally, we’ll add one more action type for changing the currently visible tod
 
 **Action creators** are exactly that—functions that create actions. It's easy to conflate the terms “action” and “action creator,” so do your best to use the proper term.
 
-In [traditional Flux](http://facebook.github.io/flux) implementations, action creators often trigger a dispatch when invoked, like so:
+In Redux action creators simply return an action:
 
 ```js
-function addTodoWithDispatch(text) {
-  const action = {
+function addTodo(text) {
+  return {
     type: ADD_TODO,
     text
   }
-  dispatch(action)
 }
 ```
 
-By contrast, in Redux action creators simply return an action:
+This makes them portable and easy to test.
+
+In Flux [traditional Flux](http://facebook.github.io/flux) action creators often trigger a dispatch when invoked, like so:
 
 ```js
-function addTodo(text) {
-  return {
+function addTodoWithDispatch(text) {
+  const action = {
     type: ADD_TODO,
     text
   }
+  dispatch(action)
 }
 ```
 
-This makes them more portable and easier to test. To actually initiate a dispatch, pass the result to the `dispatch()` function:
+In Redux to actually initiate a dispatch, pass the result to the `dispatch()` function:
```

Comment:
I would highlight: In Redux this is not the case! On the contrary, to actuallly [...]